spec/features/user_sees_activities_page_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.